pages/design/test-cases.md

```diff
@@ -41,6 +41,8 @@ In order for the mapping of implementations to the rules to work, test cases nee
 
 Related to this, failed test cases should only fail the accessibility requirements for one reason. A failed test case for 4.1.2 Name, Role, Value should not have issues with both the accessible name, and the role. If the rule is about accessible names, the role and value must be correct.
 
+Because mapping of implementations makes little difference between inapplicable and passed results, it is often better to have an inapplicable test case that "looks bad" but is actually satisfying the accessibility requirements. That way, tools that don't implement the Applicability correctly would fail this test case. For example, when testing [1.4.3 Contrast (minimum)](https://www.w3.org/TR/WCAG21/#contrast-minimum), a rule may be inapplicable to disabled button (since this matches the second exception in the Success Criterion); an inapplicable test case for this condition is better if it includes a disabled button whose text has a 2.0:1 contrast ratio with its background, than if it includes a disabled button with a 5.0:1 contrast ratio (in this second case, a tool that doesn't ignore disabled buttons would pass the test case and can still claim a correct implementation of the rule). Depending on the part of Applicability which a case tests, it is not always possible to have an inapplicable example that "looks bad".
```

Comment:
```suggestion
Because mapping of implementations makes little difference between inapplicable and passed results, it is often better to have an inapplicable test case that "looks bad" but is actually satisfying the accessibility requirements. That way, tools that don't implement the Applicability correctly would fail this test case. For example, when testing [1.4.3 Contrast (minimum)](https://www.w3.org/TR/WCAG21/#contrast-minimum), a rule may be inapplicable to disabled buttons (since this matches the second exception in the Success Criterion); an inapplicable test case for this condition is better if it includes a disabled button whose text has a 2.0:1 contrast ratio with its background, than if it includes a disabled button with a 5.0:1 contrast ratio (in this second case, a tool that doesn't ignore disabled buttons would pass the test case and can still claim a correct implementation of the rule). Depending on the part of Applicability which a case tests, it is not always possible to have an inapplicable example that "looks bad".
```